client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html

```diff
@@ -709,8 +709,8 @@
                         <label class="control-label"
                                data-translate="telephony_service_contact_status">
                         </label>
-                        <p class="form-control-static"
-                           data-ng-bind="'telephony_service_contact_sync' + (ServiceContactCtrl.directoryForm.status === 'todo' ? '_pending' : '') | translate">
+                        <p class="form-control-static">
```

Comment:
Why not prefer using the `ngBind` directive?

```suggestion
                        <p class="form-control-static" data-ng-bind=":: ServiceContactCtrl.getStatusLabel()">
```